theforeman.org/pipelines/release/source/foreman-installer.groovy

```diff
@@ -22,6 +22,11 @@ pipeline {
                 run_test(ruby: '2.5', puppet: '6.3')
             }
         }
+        stage("test ruby 2.7 & puppet 6") {
+            steps {
+                run_test(ruby: '2.7', puppet: '6.3')
```

Comment:
Perhaps it should be Puppet 7 since that ships with Ruby 2.7?

Reply:
I thought the ruby version was supposed to reflect the installer Ruby runtime, and puppet version should be the one we most often use to catch the most issues. I can add two new entries:  2.7+P7, 2.7+P6

Reply:
I don't really think that's worth it and doubt it can be done. Puppet is installed as a gem and thus runs under the Ruby env.

I have it on my agenda to support Puppet 7 in all modules and I hope to switch nightly to it soon after.